lang/src/org/partiql/lang/eval/EvaluatingCompiler.kt

```diff
@@ -1998,6 +2025,92 @@ internal class EvaluatingCompiler(
             GroupKeyExprValue(valueFactory.ion, keyValues.asSequence(), uniqueNames)
         }
 
+    private fun compileOrderByExpression(sortSpecs: List<PartiqlAst.SortSpec>): List<CompiledOrderByItem> =
+        sortSpecs.map {
+            it.orderingSpec
+                ?: errNoContext(
+                    "SortSpec.orderingSpec was not specified",
+                    errorCode = ErrorCode.INTERNAL_ERROR,
+                    internal = true
+                )
+
+            it.nullsSpec
+                ?: errNoContext(
+                    "SortSpec.nulls was not specified",
```

Comment:
typo: should say `SortSpec.nullsSpec`

```suggestion
                    "SortSpec.nullsSpec was not specified",
```